README.md

```
@@ -51,25 +42,23 @@ WujiHand QT Host Computer is a cross-platform graphical user interface applicati
 
    ```bash
-   # Simply run WujiHand_QT.exe
-   WujiHand_QT.exe
+   # Simply run wujihand-qt-hmi_v1.1.0.exe
+   wujihand-qt-hmi_v1.1.0.exe
    ```
 
 ### Linux Installation
 
-1. **Using Installation Script**
+1. **Extract and Run **
```

markdownlint-cli2 (0.18.1)

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)